docs/argo-server-sso.md

```diff
@@ -188,6 +188,25 @@ sso:
   userInfoPath: /oauth2/v1/userinfo
 ```
 
+## Filter groups
+
+> v3.5 and after
+
+```yaml
+sso:
+    # Specify a list of regular expressions to filter the groups returned by the OIDC provider.
```

Comment:
```suggestion
    # Specify a list of regular expressions to filter the groups returned by the OIDC provider.
    # A logical "OR" is used between each regex
```

Very important to specify the OR condition as it is otherwise ambiguous